pennylane_qiskit/qiskit_device.py

```diff
@@ -58,7 +58,7 @@ def z_eigs(n):
         n (int): number of wires
```

Comment:
suggest changing name to `pauli_eigs` to make implicit assumptions later on more clear

Reply:
done